codes/quantum/qubits/monitored_random_circuits.yml

```diff
@@ -0,0 +1,94 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: monitored_random_circuits
+physical: bits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Monitored Random Circuits'
+
+# First reference with the code (optional).
+introduced: '\cite{10.1103/PhysRevX.9.031009}, \cite{10.1103/PhysRevB.98.205136}, \cite{arXiv:1808.06134}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'A monitored random circuit is described by a series of intermittant local projective Pauli measurements with unitary time evolution operators. When the rate of projective measurements is independently controlled by a probability parameter p, there exist two sable phases, one described by the volume-law entanglement entropy and the other described by area-law entanglement entropy. The phase transition can be understood from the perspective of quantum error correction and information scrambling, as discussed in \cite{arXiv:1903.05124}. In the volume-law phase, the random unitaries scramble the information (rotate the code words), protecting the information from the projective measurements (aka errors). When the rate of measurements surpasses a threshold \( p_c \) the information is no longer protected. The key insight offered by the quantum error correction implemented by the scrambling dynamics is that the entanglement and correlation between subsystems A, B is scrambled into non-local degrees of freedom and cannot be disentangled by local measurements below \( p < p_c \).'
```

Comment:
Ref. should be \cite{arXiv:1903.05124,arxiv:1905.05195}.  This picture in terms of scrambling is not really accurate.  Measurements also scramble (see arxiv:2004.09560) since they are non-commuting.  It is better to phrase the problem in terms of channel capacities.  The monitored circuit has a finite information capacity that decays exponentially in time for p<pc (arxiv:1905.05195), thus, starting the system in a random code can actually achieve the capacity and allow quantum information to survive for exponentially long time.